src/main/java/org/fungover/storm/config/ConfigurationManager.java

```diff
@@ -0,0 +1,59 @@
+package org.fungover.storm.config;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ConfigurationManager {
+
+    private static ConfigurationManager myConfigurationManager;
+    private static Configuration myCurrentConfiguration;
+
+    private ConfigurationManager() {
+    }
+
+    public static ConfigurationManager getInstance(){
+        if (myConfigurationManager == null)
+            myConfigurationManager = new ConfigurationManager();
+        return myConfigurationManager;
+    }
+
+    public void readConfigurationFile(String filePath) {
+        FileReader fileReader;
+        try {
+            fileReader = new FileReader(filePath);
```

Comment:
This can be implemented using a try(FileReader fileReader = new FileReader(filePath)){}